.github/dependabot.yml

```diff
@@ -9,7 +9,7 @@ updates:
   - package-ecosystem: pip
     directory: /
     schedule:
-      interval: daily
+      interval: monthly
       time: "08:30"
       timezone: America/Santiago
```

Comment:
From [TECHINFRA-111](https://cordada.aha.io/features/TECHINFRA-111):

> Remove `schedule.time` (if present).
> Remove `schedule.timezone` (if present).

```suggestion
```

Reply:
> Created PR solving this review [fyntex/prosystem-factoring-cartera-dte-status-checker#52](https://github.com/fyntex/prosystem-factoring-cartera-dte-status-checker/pull/52)

@svillegas-cdd You probably mean https://github.com/fyntex/fd-django-accounts/pull/171.